gpii/node_modules/flowManager/src/GetGpiiKey.js

```diff
@@ -20,19 +20,18 @@ var fluid = require("infusion");
 var gpii = fluid.registerNamespace("gpii");
 
 fluid.defaults("gpii.flowManager.getGpiiKey.handler", {
-    gradeNames: ["kettle.request.http", "gpii.flowManager.sessionAware"],
+    gradeNames: ["kettle.request.http"],
```

Comment:
I don't believe this handler is required any more and as part of our general spring cleaning of unnecessary HTTP endpoints this would be a good opportunity to remove it.

Reply:
I know that it is used in several of our HTTP-based tests, but we will need to switch to do some white-box testing anyway (in order to test the proximity user A -> user B transition) so switching away from using this endpoint works well together with these other testing requirements